community/pmc-tool/collector/main.py

```diff
@@ -38,8 +38,8 @@
 
 """This file contains logic for interfacing with github and fetching PR and issue data."""
 
-import requests
 from db import create_db_connection, create_pull_requests_table, insert_prs_to_db, fetch_pull_requests
+from gh import query_prs
 
 # Set up authentication with access token
 with open('token.txt', 'r') as file:
```

Comment:
this code, and the `headers=...` code should be in gh.py. Can you move it there please? - perhaps in a new method called `_load_token`